modules/jooby-validation/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <parent>
+        <groupId>io.jooby</groupId>
+        <artifactId>modules</artifactId>
+        <version>3.2.10-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>jooby-validation</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.jooby</groupId>
+            <artifactId>jooby</artifactId>
+            <version>${jooby.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
@kliushnichenko @agentgt we can probably add the dependency on `jooby` (core). We have the inject already. This also means remove/delete this new module and move the ValidationHelper to a jooby core package.

Reply:
The idea was to keep the core free of any optional dependencies, since not every application needs/uses validation. So, it might be better to avoid adding dependencies that are rarely used.

`jakarta-inject` is tiny comparing to `validation`, and DI I believe is much more popular.
Keep in mind except `ValidationHelper` there will be ExceptionHandler added as well to the core.

Do you still want to have it in the core? You are the boss, just want to make everything clear

Reply:
> jakarta-inject is tiny comparing to validation

good point. Let's keep the new module, but let's rename to `jooby-jakarta-validation`